database/mongodb.go

```diff
@@ -13,9 +14,9 @@ import (
 // host: 127.0.0.1
 // port: 27017
 // database:
-// username:
-// password:
-// authdb:
+// username: nil, means no auth is needed.
+// password: nil
+// authdb: nil
```

Comment:
add `empty_password: true` key for instead that. `nil` is not a good idea.

https://github.com/docker-library/docs/tree/master/mysql#mysql_allow_empty_password